src/js/components/ProductDetails.svelte

```diff
@@ -21,7 +21,12 @@
     {/if}
 
     <h2 class="divider" id="productNameWithoutBrand">{product.NameWithoutBrand}</h2>
-    <img id="productImage" class="divider" src="{product.Image}" alt="{product.Name}" />
+    <picture>
+        <source media="(max-width: 213px)" srcset="{product.Images.small}" />
```

Comment:
This is really good. I think we should adjust these media values though, the max-width here represents the width of the whole screen, which pretty much never will get as small as 200px even on a phone.

Values around this range were working for me when I messed with it:
![image](https://github.com/levi-wj/sleepoutside-team-4/assets/37312664/7cae670a-4714-46e7-9e27-01ff4a148a32)